docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -786,8 +775,8 @@ Next, generate a flame graph using the data captured via `perf record`:
 
 [source,bash]
 ----
-$ perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded
-$ ${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
+perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded
+${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
```

Comment:
For even easier copying, we could convert these two commands into a single one. E.g. 

```
perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded \ 
    && ${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
```

Reply:
Even better indeed :)